### ISSUE TITLE
കാൻവാസിലെ ടെക്സ്റ്റ്ബോക്സിലുള്ള ടൈപ്പിങ്ങ് പ്രശ്നങ്ങൾ

### ISSUE THREAD

**Issue by balasankarc** [meenaviyal/comicmachine#5]
ക്ലിൿ മീയിൽ (കാൻവാസിൽ ചേർത്ത ഏതൊരു ടെക്സ്റ്റ് ബോക്സിലും) സ്വനലേഖ ഉപയോഗിച്ച് ടൈപ്പ് ചെയ്യുമ്പോൾ പല അക്ഷരങ്ങളും (എനിക്ക് തോന്നുന്നത് കമ്മിറ്റ് ചെയ്യേണ്ട - Using ZWJ/ZWNJ - അക്ഷരങ്ങൾക്കാണു് പ്രശ്നമെന്നാ. അതായത് ചില്ലക്ഷരങ്ങൾ, അനുസ്വാരം മുതലായവ) തെളിയുന്നില്ല. മൊഴി ഉപയോഗിക്കുമ്പോളും "വർണ്ണം ", "കർമ്മം " എന്നൊക്കെ അടിക്കുമ്പോൾ അവസാനത്തെ അനുസ്വാരം തെളിയുന്നില്ല.

വലതുവശത്തെ സൈഡ്ബാറിലുള്ള ടെക്സ്റ്റ് ബോക്സിനു് ഈ പ്രശ്നമൊന്നുമില്ല. ടെക്സ്റ്റ് ബോക്സ് കാൻവാസിലെത്തി കഴിഞ്ഞാണു് പ്രശ്നം.

Debian Sid / Firefox 47.0.1

**Comment by hfactor** [comment]
Let's remove that click me text. Let the canvas be empty. @zmattra

**Comment by balasankarc** [comment]
പക്ഷേ ഇൻസേർട്ട് ചെയ്തുകഴിയുമ്പോൾ ഏത് ടെക്സ്റ്റ്ബോക്സും ഇങ്ങനെ തന്നെയാ പെരുമാറുന്നത്. ക്ലിൿ മീ മാത്രമല്ല. 

തൽക്കാലം എഡിറ്റിങ്ങ് പറ്റില്ലെന്നേ ഉള്ളൂ. തെറ്റുവന്നാൽ ടെക്സ്റ്റ് ഡിലീറ്റിയിട്ട് വേറെ ഉണ്ടാക്കണം.

@hfactor @zmattra

**Comment by hfactor** [comment]
Yes. That's how it (library) works.

**Comment by balasankarc** [comment]
Oh. Ok. We can live with that. Remove the click me then. It makes more trouble than its worth. :)

**Comment by lebmatter** [comment]
@balasankarc Can you sent screenshots of this issue? We could not reproduce this.
@stultus can you check this?

**Comment by balasankarc** [comment]
@stultus @zmattra See this screencast. I believe this is more efficient than a screenshot.

http://balasankarc.in/bug.mkv

**Comment by lebmatter** [comment]
This is a fabricjs issue.
Even it was not working for Google Transliteration tool
I think we need to raise an issue in upstream.

**Comment by stultus** [comment]
I don't think that it is a fabricjs issue. because it works fine for me in latest firefox. 
issue should be with ibus.  (mine is latest ibus from aur)

**Comment by balasankarc** [comment]
I updated to latest IBus and problem still exists.
